src/clj_test_containers/core.clj

```diff
@@ -29,6 +34,24 @@
      :env-vars (.getEnvMap container)
      :host (.getHost container)}))
 
+(defn create-from-docker-file
+  [{:keys [exposed-ports env-vars command docker-file]
+    :or {docker-file "Dockerfile"}}]
+  (let [docker-image (ImageFromDockerfile.)
```

Comment:
`docker-image` is defined twice in the `let`. Wouldn't it be possible to include the constructor into the threading macro, also?